convert2rhel/utils.py

```diff
@@ -247,11 +247,17 @@ def ask_to_continue():
 
 def prompt_user(question, password=False):
     color_question = Color.BOLD + question + Color.END
-    if password:
-        response = getpass.getpass(color_question)
-    else:
-        response = moves.input(color_question)
-    loggerinst.info("\n")
+
+    while True:
+        if password:
+            response = getpass.getpass(color_question)
+        else:
+            response = moves.input(color_question)
+        loggerinst.info("\n")
+
+        if response != "" or len(response) > 0:
```

Comment:
Is there a time when the empty string test and the length test would return different values?

Checking whether something is Falsey (like the following) is more pythonic but I don't know if there's a reason that you are avoiding it? 

``` python
if not response:
    break
```

Reply:
Right! I think I just left this piece of code for when I was testing out. A simple 

```python
if response: 
   break
```

would work properly!